sda-commons-server-prometheus/README.md

```diff
@@ -34,6 +34,97 @@ Default metrics that are provided at `/metrics/prometheus`:
 *) A filter that extracts the consumer from the HTTP headers should add `Consumer-Name` to the request properties. That
    filter is not part of the `PrometheusBundle`.
 
+
+Kafka internal producer metrics have been exposed
+
+| Metric name                                                | Labels                 | Description                                                                                                                            | Source |
+|------------------------------------------------------------|------------------------|----------------------------------------------------------------------------------------------------------------------------------------|--------|
+| `kafka_producer_batch_split_rate`                          | `client.id`            | The average number of batch splits per second                                                                                          |        |
```

Comment:
Is it possible to write a test that validates all these metrics are being sent, or it is not possible / too hard in a test context?

Reply:
That would be nice to ensure backward compatibility. Or at least notice when something changes.

Reply:
Test added to KafkaPrometheusMonitoringIT